tests/unittests/CoreGraphics.Drawing/CGPathDrawingTests.cpp

```diff
@@ -579,7 +579,69 @@ DRAW_TEST_F(CGPath, FillArcsSimple, UIKitMimicTest<>) {
     CGPathRelease(thepath);
 }
 
-DRAW_TEST_F(CGPath, FillArcsComplex, UIKitMimicTest<>) {
+DRAW_TEST_F(CGPath, BoundingBoxes, UIKitMimicTest<>) {
```

Comment:
there are exact duplicates of this tests below, this would be a good test to be parameterized.